src/igvfd/audit/in_vitro_system.py

```diff
@@ -43,14 +43,14 @@ def audit_cell_fate_change_treatments_purpose(value, system):
                     f'has cell_fate_change_treatment {audit_link(path_to_text(treatment["@id"]), treatment["@id"])} '
                     f'that has purpose {treatment["purpose"]}.'
                 )
-                yield AuditFailure('inconsistent cell_fate_change_treatments treatment purpose', detail, level='ERROR')
+                yield AuditFailure('inconsistent treatment purpose', detail, level='WARNING')
 
 
 @audit_checker('InVitroSystem', frame='embedded')
 def audit_cell_fate_change_protocol_document_type(value, system):
     '''
-        audit_detail: A document linked to in an in vitro system's cell_fate_change_protocol must be of document_type cell fate change protocol.
-        audit_category: inconsistent cell_fate_change_protocol document type
+        audit_detail: A document linked to as a cell fate change protocol is expected to have a cell fate change protocol document type.
```

Comment:
A document listed as a cell fate change protocol is expected to have a cell fate change protocol document type.??

Reply:
fixed